torch/_functorch/aot_autograd.py

```diff
@@ -1806,6 +1832,7 @@ def merge_view_inputs(
 ) -> Tuple[List[Any], Optional[List[Union[int, Tuple[int, torch.Tensor]]]]]:
     assert len(fwd_inputs) == len(mutated_input_info)
     storage_ref_to_idx: Dict[StorageWeakRef, List[int]] = collections.defaultdict(list)
+    return fwd_inputs, None
```

Comment:
🤔

Reply:
This is a Certified @bdhirsh + I plan lol.

This is known from the talk as "Handling mutations on an input that aliases other inputs"